From 308b33b7e4a34d6622e7bab0797ed1c094a532b7 Mon Sep 17 00:00:00 2001
From: Jakub Filak <jfilak@redhat.com>
Date: Fri, 27 Feb 2015 14:44:22 +0100
Subject: [LIBREPORT PATCH] ureport: rename its event to submit_uReport

report-gtk displays all events prefixed with 'report' as available reports, but
we don't want to show uReport event there.

Signed-off-by: Jakub Filak <jfilak@redhat.com>
---
 po/POTFILES.in                       |  2 +-
 src/plugins/Makefile.am              |  4 ++--
 src/plugins/report_RHTSupport.xml.in |  2 +-
 src/plugins/report_uReport.xml.in    | 46 ------------------------------------
 src/plugins/submit_uReport.xml.in    | 46 ++++++++++++++++++++++++++++++++++++
 5 files changed, 50 insertions(+), 50 deletions(-)
 delete mode 100644 src/plugins/report_uReport.xml.in
 create mode 100644 src/plugins/submit_uReport.xml.in

diff --git a/po/POTFILES.in b/po/POTFILES.in
index 16a938e..a7760d0 100644
--- a/po/POTFILES.in
+++ b/po/POTFILES.in
@@ -29,7 +29,7 @@ src/plugins/report_RHTSupport.xml.in
 src/plugins/report_RHTSupportAttach.xml.in
 src/plugins/report_Tarball.xml.in
 src/plugins/report_Uploader.xml.in
-src/plugins/report_uReport.xml.in
+src/plugins/submit_uReport.xml.in
 src/plugins/reporter-bugzilla.c
 src/plugins/reporter-kerneloops.c
 src/plugins/reporter-mailx.c
diff --git a/src/plugins/Makefile.am b/src/plugins/Makefile.am
index e780265..e9e626e 100644
--- a/src/plugins/Makefile.am
+++ b/src/plugins/Makefile.am
@@ -33,7 +33,7 @@ dist_events_DATA = \
     report_Kerneloops.xml \
     report_Tarball.xml \
     report_Uploader.xml \
-    report_uReport.xml
+    submit_uReport.xml
 
 @INTLTOOL_XML_RULE@
 
@@ -58,7 +58,7 @@ EXTRA_DIST = \
     report_Kerneloops.xml.in \
     report_Tarball.xml.in \
     report_Uploader.xml.in \
-    report_uReport.xml.in
+    submit_uReport.xml.in
 
 $(DESTDIR)/$(DEBUG_INFO_DIR):
 	$(mkdir_p) '$@'
diff --git a/src/plugins/report_RHTSupport.xml.in b/src/plugins/report_RHTSupport.xml.in
index aa3e886..b4e68f2 100644
--- a/src/plugins/report_RHTSupport.xml.in
+++ b/src/plugins/report_RHTSupport.xml.in
@@ -11,7 +11,7 @@
     <minimal-rating>0</minimal-rating>
 
     <options>
-        <import-event-options event="report_uReport"/>
+        <import-event-options event="submit_uReport"/>
 
         <option type="text" name="RHTSupport_Login">
             <_label>Username</_label>
diff --git a/src/plugins/report_uReport.xml.in b/src/plugins/report_uReport.xml.in
deleted file mode 100644
index 6a6c45a..0000000
--- a/src/plugins/report_uReport.xml.in
+++ /dev/null
@@ -1,46 +0,0 @@
-<?xml version="1.0" encoding="UTF-8" ?>
-<event>
-    <_name>uReport</_name>
-    <_description>Sends ureports to FAF server</_description>
-
-    <requires-items>analyzer,reason,executable,pkg_epoch,pkg_name,pkg_version,pkg_release,pkg_arch,os_release,architecture,core_backtrace</requires-items>
-    <gui-review-elements>no</gui-review-elements>
-
-    <options>
-        <import-event-options event="watch_Bugzilla"/>
-
-        <option type="text" name="uReport_URL">
-            <_label>uReport Server URL</_label>
-            <allow-empty>no</allow-empty>
-            <_description>Address of uReport webservice</_description>
-            <default-value>http://bug-report.itos.redhat.com</default-value>
-        </option>
-        <option type="bool" name="uReport_WatchReportedBugs">
-            <_label>Watch already reported bugs</_label>
-            <_note-html>Adds your Bugzilla login to CC List of already reported bugs.</_note-html>
-            <default-value>yes</default-value>
-        </option>
-        <option type="text" name="uReport_ContactEmail">
-            <_label>Contact email address</_label>
-            <allow-empty>yes</allow-empty>
-            <_description>Email address that can be used by ABRT server to inform you about news and updates</_description>
-        </option>
-        <option type="bool" name="uReport_SSLVerify">
-            <_label>Verify SSL</_label>
-            <_description>Check SSL key validity</_description>
-            <default-value>yes</default-value>
-        </option>
-        <advanced-options>
-            <option type="text" name="http_proxy">
-                <_label>HTTP Proxy</_label>
-                <allow-empty>yes</allow-empty>
-                <_note-html>Sets the proxy server to use for HTTP</_note-html>
-            </option>
-            <option type="text" name="HTTPS_PROXY">
-                <_label>HTTPS Proxy</_label>
-                <allow-empty>yes</allow-empty>
-                <_note-html>Sets the proxy server to use for HTTPS</_note-html>
-            </option>
-        </advanced-options>
-    </options>
-</event>
diff --git a/src/plugins/submit_uReport.xml.in b/src/plugins/submit_uReport.xml.in
new file mode 100644
index 0000000..1aadeaf
--- /dev/null
+++ b/src/plugins/submit_uReport.xml.in
@@ -0,0 +1,46 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<event>
+    <_name>uReport</_name>
+    <_description>Sends micro-reports to ABRT server</_description>
+
+    <requires-items>analyzer,reason,executable,pkg_epoch,pkg_name,pkg_version,pkg_release,pkg_arch,os_release,architecture,core_backtrace</requires-items>
+    <gui-review-elements>no</gui-review-elements>
+
+    <options>
+        <import-event-options event="watch_Bugzilla"/>
+
+        <option type="text" name="uReport_URL">
+            <_label>uReport Server URL</_label>
+            <allow-empty>no</allow-empty>
+            <_description>Address of uReport webservice</_description>
+            <default-value>http://bug-report.itos.redhat.com</default-value>
+        </option>
+        <option type="bool" name="uReport_WatchReportedBugs">
+            <_label>Watch already reported bugs</_label>
+            <_note-html>Adds your Bugzilla login to CC List of already reported bugs.</_note-html>
+            <default-value>yes</default-value>
+        </option>
+        <option type="text" name="uReport_ContactEmail">
+            <_label>Contact email address</_label>
+            <allow-empty>yes</allow-empty>
+            <_description>Email address that can be used by ABRT server to inform you about news and updates</_description>
+        </option>
+        <option type="bool" name="uReport_SSLVerify">
+            <_label>Verify SSL</_label>
+            <_description>Check SSL key validity</_description>
+            <default-value>yes</default-value>
+        </option>
+        <advanced-options>
+            <option type="text" name="http_proxy">
+                <_label>HTTP Proxy</_label>
+                <allow-empty>yes</allow-empty>
+                <_note-html>Sets the proxy server to use for HTTP</_note-html>
+            </option>
+            <option type="text" name="HTTPS_PROXY">
+                <_label>HTTPS Proxy</_label>
+                <allow-empty>yes</allow-empty>
+                <_note-html>Sets the proxy server to use for HTTPS</_note-html>
+            </option>
+        </advanced-options>
+    </options>
+</event>
-- 
1.8.3.1

